kedro-org/kedro-viz
# Release v6.5.0

## Description

This is a minor release since we are adding support for Python 3.11 and removing upper-bounds on Python. 

## Major features and improvements

- Add support for Python 3.11 (#1502)

## Bug fixes and other changes

- Fix to search for a '<lambda' Python function in the sidebar. (#1497)
- Remove python upper-bound requirements and add KedroVizPythonVersion warning. (#1506)
- Add favicon to Kedro-Viz. (#1509)

## Development notes



## QA notes

View the Gitpod link and make sure all is good. Please also check the update reminders section. 

## Checklist



<a href="https://gitpod.io/#https://github.com/kedro-org/kedro-viz/pull/1513"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
